master/buildbot/test/unit/test_scripts_checkconfig.py

```diff
@@ -103,7 +103,7 @@ def test_failure_ImportError(self):
                 """)
         self.do_test_load(config=config,
                           stderr_re=re.compile(
-                              'No module named test_scripts_checkconfig_does_not_exist'))
+                              "No module named (')?test_scripts_checkconfig_does_not_exist(')?"))
```

Comment:
I believe following should be enough:

    "No module named '?test_scripts_checkconfig_does_not_exist'?"

Please add a comment stating thaht this is for python implementation difference